onnx_caffe2/frontend.py

```diff
@@ -238,6 +239,33 @@ def mk():
         return self._add(name, mk())
 
 
+def caffe2_init_net_to_initializers(init_net):
+    initializers = []
+    for init_op in init_net.op:
+        assert not init_op.input
+        data_type_map = {
+            'GivenTensorFill': (onnx_pb2.TensorProto.FLOAT, 'floats', np.float32),
+            'GivenTensorInt64Fill': (onnx_pb2.TensorProto.INT64, 'ints', np.int64),
+            'GivenTensorIntFill': (onnx_pb2.TensorProto.INT32, 'ints', np.int32),
+            'GivenTensorBoolFill': (onnx_pb2.TensorProto.BOOL, 'ints', np.int32),
```

Comment:
Could you refer to https://github.com/bddppq/onnx-caffe2/blob/fdece9a1015f1a81a66efaebd731ed87223eb626/onnx_caffe2/backend.py#L173 for types of tensor and FillOps? There are more types, and also a GivenTensorStringFill as well.

Reply:
It's mapping different things. Here I want to map from "Given*Fill" ops to (TensorProto types, np types) pair.
I was intentionally ignoring GivenTensorString operator since the `raw_data` field can not be used to store string types. However, on second thought, we don't need all initializers to be using `raw_data`, so I think I should add support for GivenTensorString here. Thanks for pointing it out!

Reply:
Looks like the types in create_constant does not fully cover all the types in fill_values, could you fix that as well?

Reply:
sure, just created #23 for completing it.